spec/beaker-pe/install/pe_utils_spec.rb

```diff
@@ -1972,9 +1972,9 @@ def slice_installer_options(host)
   end
 
   describe 'simple_monolithic_install' do
-    let(:monolithic) { make_host('monolithic', :pe_ver => '2016.4', :platform => 'el-7-x86_64', :roles => [ 'master', 'database', 'dashboard' ]) }
-    let(:el_agent) { make_host('agent', :pe_ver => '2016.4', :platform => 'el-7-x86_64', :roles => ['frictionless']) }
-    let(:deb_agent) { make_host('agent', :pe_ver => '2016.4', :platform => 'debian-7-x86_64', :roles => ['frictionless']) }
+    let(:monolithic) { make_host('monolithic', :pe_ver => '2016.4', :platform => 'el-7-x86_64', :packaging_platform => 'el-7-x86_64', :roles => [ 'master', 'database', 'dashboard' ]) }
+    let(:el_agent) { make_host('agent', :pe_ver => '2016.4', :platform => 'el-7-x86_64', :packaging_platform => 'el-7x-86_64', :roles => ['frictionless']) }
```

Comment:
Looks like a typo here

Reply:
That's weird I just ran these locally.

Reply:
Oh, that is a typo, but not why specs are failing. 2.4.4 is broken around activesupport

Reply:
😰

Reply:
ok both issues should be fixed